src/style.scss

```diff
@@ -1,3 +1,44 @@
-.App {
-    background: green;
+$base-color: #FFCD1B;
+$black-color: #202020;
+$gray-color: #606060;
+$background-color: #e5e5e5;
+$white-color: #ffffff;
+
+
+.button {
+    border: none;
+    cursor: pointer;
+     &--large {
+        background: $base-color;
+        color: $black-color;
+        padding: 6px;
+        width: 50px;
+        height: 50px;
+        & img{
```

Comment:
Icon components are injected into DOM as embedded SVG elements, so you need to use `& svg` if you want to style them.